pages/faq.tsx

```diff
@@ -107,7 +107,7 @@ function FAQ({}: Props) {
         </FAQEntry>
         <FAQEntry title="I can't to open Ferdium after downloading. What should I do?">
           <>
-            This is because the releases aren't yet signed. We are still working on getting an Apple developer license and a certificate for Windows.
+            This is because the releases aren&apost yet signed. We are still working on getting an Apple developer license and a certificate for Windows.
```

Comment:
Is this change necessary?

Reply:
`next lint` asks for it, it's why the [latest commit deploy](https://github.com/ferdium/website/runs/6495657554?check_suite_focus=true) failed

Reply:
oops